docs/index.md

```diff
@@ -25,17 +25,17 @@ Mocha is a feature-rich JavaScript test framework running on [Node.js](https://n
 - [maps uncaught exceptions to the correct test case](#browser-specific-methods)
 - [async test timeout support](#delayed-root-suite)
 - [test retry support](#retry-tests)
-- [test-specific timeouts](#test-level) 
+- [test-specific timeouts](#test-level)
```

Comment:
looks like the whitespace changes crept back in.  that other PR has been merged, so if there are no conflicts then this is fine.  please rebase onto `master` if you haven't done so yet